src/console_reporter.cc

```diff
@@ -100,7 +115,22 @@ void ConsoleReporter::PrintRunData(const Run& result) {
   ColorPrintf(COLOR_GREEN, "%-*s ",
```

Comment:
consider different colors for the complexity report. they get lost in the output otherwise.

also, i don't think there's any need to output for both time and cpu, but you may think different.

you can suppress the iteration count too.